src/django/api/templates/mail/claim_facility_approval_body.html

```diff
@@ -21,16 +21,17 @@
                 Facility URL: {{ facility_url }}
             </li>
         </ul>
-        {% if approval_reason|length %}
         <p>
             Here's the reason for the approval:
         </p>
         <p>
-            {{ approval_reason }}
+            You are approved! You can now complete your profile, adding business information
```

Comment:
This change matches what was requested in the issue, but I want to confirm that there was some discussion about not having the reason entered into the approval form appear in the email. It reads a little strangely now that there is a line that says "Here's the reason" and then no reason is given. 

The approval form now incorrectly says that the content will be mailed to the claimant.

<img width="577" alt="Screen Shot 2019-06-28 at 10 12 53 AM" src="https://user-images.githubusercontent.com/17363/60359239-578fef80-998d-11e9-8a1a-97ca5cf130a3.png">

Reply:
I'll add a comment on the original issue to get this confirmed.